tindergames/src/main/java/org/teamcifo/logic/LogIn.java

```diff
@@ -0,0 +1,134 @@
+package org.teamcifo.logic;
+
+import org.teamcifo.domain.User;
+import java.util.Scanner;
+
+public class LogIn {
+    private UserManager userManager;
+    private Scanner scanner;
+
+    public LogIn(UserManager userManager) {
+        //this.authService = authService;
+        this.userManager= userManager;
+        this.scanner = new Scanner(System.in);
+    }
```

Comment:
`LogIn` class shouldn't instantiate a `UserManager` object because `UserManager` should be defined as an Static Class. `LogIn` methods should invoke `UserManager` static methods externally, they shouldn't have this level of coupling and dependency.

Reply:
To review in another task